src/power/PowerComponentWin.cpp

```diff
@@ -13,3 +15,52 @@ void PowerComponentWin::doEnableScreensaver()
 {
   SetThreadExecutionState(ES_CONTINUOUS);
 }
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+bool PowerComponentWin::Suspend()
+{
+
+  return SetSuspendState(false, true, false) == TRUE;
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+bool PowerComponentWin::Reboot()
+{
+  if (!WinUtils::getPowerManagementPrivileges())
+  {
+    QLOG_ERROR() << "Failed to obtain power management, priviledges";
+    return false;
+  }
+
+  if (QSysInfo::WindowsVersion >= QSysInfo::WV_WINDOWS8)
+  {
+    return InitiateShutdownW(NULL, NULL, 0, SHUTDOWN_INSTALL_UPDATES | SHUTDOWN_RESTART,
```

Comment:
I'm likely going daft - but I don't see the diff here. Aren't these the same calls/flags?